packages/amplify-e2e-core/src/init/overrideStack.ts

```diff
@@ -35,3 +35,20 @@ export function amplifyOverrideAuth(cwd: string, settings: {}) {
       });
   });
 }
+
+export function amplifyOverrideApi(cwd: string, settings: any) {
+  return new Promise((resolve, reject) => {
+    const args = ['override', 'api'];
+    const chain = spawn(getCLIPath(), args, { cwd, stripColors: true });
+    if (settings.isMigratedProject === true) {
+      chain.wait('Do you want to edit override.ts file now?').sendConfirmNo().sendEof();
+    }
+    chain.run((err: Error) => {
```

Comment:
nit: would be nice to use chain.runAsync() instead

Reply:
ill keep it same for now as all tests are using this